CMakeLists.txt

```diff
@@ -223,17 +223,17 @@ add_subdirectory(docs)
 
 # DESTINATION gets the previous path and appends the argument --> /opt/local/GrPPI
```

Comment:
Because there are several ways to make an installation, we have in the comments several examples of the different options. This is an explanation of how DESTINATION works, but we can remove it.